website/docs/docs/collaborate/explore-projects.md

```diff
@@ -130,16 +158,11 @@ You can view the definition and latest run results of any resource in your proje
 
 The details (metadata) available to you depends on the resource’s type, its definition, and the [commands](/docs/deploy/job-commands) that run within jobs in the production environment. 
 
-
-<LoomVideo id='cefbb113427e45fd8106607476ec0676' />
-
-### Example of model details
-
-An example of the details you might get for a model:
+<expandable alt_header="What details are available for a model?">
 
 - Status bar (below the page title) &mdash; Information on the last time the model ran, whether the run was successful, how the data is materialized, number of rows, and the size of the model. 
```

Comment:
```suggestion
- **Status bar** (below the page title) &mdash; Information on the last time the model ran, whether the run was successful, how the data is materialized, number of rows, and the size of the model. 
```

Reply:
this was deliberate actually. there is no `Status bar` UI label which is why it's not bolded. it's just referring to the area between the page title (ie resource name) and start of the tabs **General**, **Code**, etc.

Reply:
fair point, i'll fold in the bolding. thanks!